### PR TITLE
[IT-3021] Prioritize the RECOVER code

### PR DESCRIPTION
**Pull request by jesusaurus**
Prioritize the RECOVER code so that it is listed near the top of the lambda output, ensuring that it is available in the Service Catalog drop-down menu.

Also update mips-api parameters to match name changes happening in the dependent PR.

Depends on Sage-Bionetworks-IT/lambda-mips-api#21

### CODE REVIEW

File: sceptre/scipool/config/develop/sc-tag-options.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   - "develop/sc-portfolio-scheduled-jobs.yaml"
 sceptre_user_data:
   CostCenters: !from_json
-    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true'
+    - !request 'https://mips-api.finops.sageit.org/tags?only_codes=101300,101400,101600,112501,119400,121700,120100,122000,122100,122300,122500,312000,314900,401900,506800,613500,990100,990300,990400&enable_other_code=true'
```
Comment (zaro0508) [inline sceptre/scipool/config/develop/sc-tag-options.yaml:14]: i'm not sure i like this idea because i think enumerating every code we want on the list is difficult to maintain which is really no better than a static list.  what if one of these codes becomes inactive?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: sceptre/scipool/config/develop/sc-tag-options.yaml (hunk 1 above)
Comment (brucehoff) [reply sceptre/scipool/config/develop/sc-tag-options.yaml:14]: @zaro0508 Can you answer @jesusaurus 's question?  I think you are saying that you'd prefer the list to be maintained somewhere else.  Can you suggest an alternative?

Comment (jesusaurus) [reply sceptre/scipool/config/develop/sc-tag-options.yaml]: I've re-worked the underlying PR to be a priority list

File: sceptre/scipool/config/develop/sc-tag-options.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ dependencies:
   - "develop/sc-portfolio-scheduled-jobs.yaml"
 sceptre_user_data:
   CostCenters: !from_json
-    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true'
+    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true&priority_codes=122500'
```
Comment (zaro0508) [inline sceptre/scipool/config/develop/sc-tag-options.yaml:14]: could you please add a comment explaining why we are doing this?  maybe just a link to the issue might be good enough

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/050-costs/_tasks.yaml [hunk 3]
```diff
@@ -42,7 +42,7 @@ CostCategoryRulesMicroservice:
   Parameters:
     AcmCertificateArn: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-sageit-finops-cert-CertificateArn']
     DnsName: "cost-rules.finops.sageit.org"
-    ChartOfAccountsURL: "https://mips-api.finops.sageit.org/accounts?enable_code_filter&disable_inactive_filter"
+    ChartOfAccountsURL: "https://mips-api.finops.sageit.org/accounts?show_inactive_codes=on"
```
Comment (zaro0508) [inline org-formation/050-costs/_tasks.yaml:45]: what is the purpose of this change?  also i don't see any documentation for `show_inactive_codes` option in the lambda-finops-cost-rules lambda docs.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/050-costs/_tasks.yaml (hunk 3 above)
Comment (jesusaurus) [reply org-formation/050-costs/_tasks.yaml:45]: We need this to match the parameter name changes happening in https://github.com/Sage-Bionetworks-IT/lambda-mips-api/pull/21

File: sceptre/scipool/config/develop/sc-tag-options.yaml (hunk 2 above)
Comment (jesusaurus) [reply sceptre/scipool/config/develop/sc-tag-options.yaml]: done

File: org-formation/050-costs/_tasks.yaml (hunk 3 above)
Comment (zaro0508) [reply org-formation/050-costs/_tasks.yaml:45]: ahh ok.

**Review by zaro0508 — APPROVED** [review]
